_build/test/Tests/Model/Request/modRequestTest.php

```diff
@@ -316,7 +316,14 @@ public function providerSanitizeRequest() {
             array('MODX [[fakeSnippet]] Tags','MODX  Tags'),
             array("MODX [[\$chunk? &property=`test`\n &across=`lines
 
-` &test=1]] Tags",'MODX  Tags'),
+                   `&test=1]] Tags",'MODX  Tags'),
+            array("Nested MODX [[test? 
+                   &ids = `[[!getids 
+                   &field=`id` 
+                   &resource=`[[+resource]]`
+                   ]]
```

Comment:
The test even works with invalid tags. A ` is missing here.

Reply:
@Jako can you submit a PR with a fix for the test?

Reply:
It makes no difference. Since the ` does not touch the regular expression for nested tags.